### DISCUSSION TITLE
Welcome to Jekyll!

### DISCUSSION THREAD

**Discussion by giscus** [uzza1hossain/uzza1hossain.github.io#1] [Announcements]
# Welcome to Jekyll!

You’ll find this post in your _posts directory. Go ahead and edit it and re-build the site to see your changes. You can rebuild the site in many different ways, but the most common way is to run jekyll serve, which launches a web server and auto-regenerates your site when a file is updated.

https://uzza1hossain.github.io/blog/welcome-to-jekyll/

**Comment by munia2020** [comment]
test comment

**Reply by uzza1hossain** [reply]
Hello
